.github/ISSUE_TEMPLATE/collection_proposal.md

```diff
@@ -3,12 +3,12 @@ name: Propose a new collection
 about: Use this ticket when you are submitting a new collection for review.
 title: 'collection: <Add your title>'
 labels: content proposal
-assignees: ''
-
+assignees: 'kaycebasques'
 ---
 
-**Author**: @[Author's GitHub username.]
-**Reviewer**: [Leave blank. web.dev team will assign a reviewer.]
+**Author**: @[Author's GitHub username]
+**SME Reviewer**: @[SME's GitHub username]
```

Comment:
What's the intent of this? Within DevRel, the SME is probably the person who wrote.

Are you expecting their LGTM in the PR? Frequently my SME reviewer has already approved the draft in Docs.